client/src/test/scala/org/http4s/client/ClientRouteTestBattery.scala

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright 2014 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package client
+
+import cats.effect._
+import cats.syntax.all._
+import fs2._
+import org.http4s.client.dsl.Http4sClientDsl
+import org.http4s.client.scaffold.ServerScaffold
+import org.http4s.client.testroutes.GetRoutes
+import org.http4s.dsl.io._
+import org.http4s.implicits._
+import org.http4s.multipart.Multipart
+import org.http4s.multipart.Part
+import org.typelevel.ci._
+
+import java.util.Arrays
+import java.util.Locale
+import scala.concurrent.duration._
+
+abstract class ClientRouteTestBattery(name: String) extends Http4sSuite with Http4sClientDsl[IO] {
```

Comment:
If I'm not wrong, this is the `0.22`-related suite.

Reply:
Yes, these files are actually misplaced.
```
[error] (coreJS / Test / test) Misplaced sources in /home/runner/work/http4s/http4s/core/src
[error] (testingJS / Test / test) Misplaced sources in /home/runner/work/http4s/http4s/testing/src
[error] (testsJS / Test / test) Misplaced sources in /home/runner/work/http4s/http4s/tests/src
[error] (ember-clientJS / Test / test) Misplaced sources in /home/runner/work/http4s/http4s/ember-client/src
[error] (clientJS / Test / test) Misplaced sources in /home/runner/work/http4s/http4s/client/src
```

Reply:
Ah, hell.  I think git didn't catch all the moved files, and I didn't catch the ones it didn't catch.